First draft

drvinceknight opened the pull request:
Also: improve the colormap for the median plots.

I'm opening this PR which will have more of my work.

I've also tidied up our db data folder: compressing everything that was there before, and copying over my `data` folder to an `axelrod-moran` folder. The `data` folder contains both the raw data  and the data that is obtained by running the cleaning srcipts (in `src`).

drvinceknight committed:
Add boxplot visualisation.

Also: improve the colormap for the median plots.

drvinceknight committed:
Add boxplot visualisation.

Also: improve the colormap for the median plots.

drvinceknight committed:
Add a table of linear regression coefficients.

This is neat (I think) as it shows numerically that the N=2 case has
very little to do with an actual population dynamic.

drvinceknight committed:
Add a table of linear regression coefficients.

This is neat (I think) as it shows numerically that the N=2 case has
very little to do with an actual population dynamic.

drvinceknight committed:
Update plots.

drvinceknight committed:
Update plots.

drvinceknight committed:
Add exact moran process to validation script.

drvinceknight committed:
Add exact moran process to validation script.

drvinceknight committed:
Prepare to run 1, N-1 case.

drvinceknight committed:
Prepare to run 1, N-1 case.

drvinceknight committed:
Revert symmetric case.

drvinceknight committed:
Revert symmetric case.

drvinceknight committed:
Remove exact from validation plot.

The run time is unreasonable.

drvinceknight committed:
Remove exact from validation plot.

The run time is unreasonable.

drvinceknight committed:
Start refactoring for different initial population.

drvinceknight committed:
Start refactoring for different initial population.

drvinceknight committed:
Merge branch 'vk' of github.com:Axelrod-Python/axelrod-moran into vk

drvinceknight committed:
Readjust to include invasion/resistance.

drvinceknight committed:
Readjust to include invasion/resistance.

drvinceknight commented:
@marcharper I've rewritten the notebook and other files to include 1, N-1. There still a lot of data missing (early days) but it's on it's way.

drvinceknight committed:
Update results.

drvinceknight committed:
Update results.

drvinceknight committed:
Add code to numerically find relative fitness.

Also remove the Noisy data from the cleaning step.

drvinceknight committed:
Add code to numerically find relative fitness.

Also remove the Noisy data from the cleaning step.

drvinceknight committed:
Change title: numerical -> empirical

drvinceknight committed:
Change title: numerical -> empirical

drvinceknight committed:
Write a final data set with all data.

drvinceknight committed:
Write a final data set with all data.

drvinceknight committed:
Add a `main.csv` data that makes all analysis easier.

drvinceknight committed:
Add a `main.csv` data that makes all analysis easier.

drvinceknight committed:
Add more data.

drvinceknight committed:
Add more data.

drvinceknight committed:
Change a number of styles of plots and writing.

Modified the format from horizontal to vertical and I also used a
correlation matrix (which was essentially what I was doing before).

drvinceknight committed:
Change a number of styles of plots and writing.

Modified the format from horizontal to vertical and I also used a
correlation matrix (which was essentially what I was doing before).

marcharper commented on tbl/summary_top_10_invade.tex:
```diff
@@ -0,0 +1,11 @@
+\midrule
+        CollectiveStrategy &         0.119 &            \(\infty\) &      False \\
+                  Predator &         0.114 &             9 &      False \\
+   BackStabber: ('D', 'D') &         0.113 &            \(\infty\) &      False \\
```
We might consider not using these two strategies since they use knowledge of the tournament length

drvinceknight replied:
Yeah: good shout. I'll adjust that.

marcharper commented on tbl/summary_top_10_resist.tex:
```diff
@@ -0,0 +1,11 @@
+\toprule
+             Player &  Median $p_{N-1}$ &  Memory Depth & Stochastic \\
+\midrule
+ CollectiveStrategy &             0.119 &            \(\infty\) &      False \\
```
These handshakers make sense

drvinceknight replied:
:+1: :)

marcharper commented on tbl/summary_top_2_invade.tex:
```diff
@@ -0,0 +1,11 @@
+\toprule
+                                    Player &  Median $p_1$ &  Memory Depth & Stochastic \\
+\midrule
+        Meta Winner Memory One: 31 players &         0.587 &            \(\infty\) &       True \\
```
MWMO is surprising, the ZDs make sense

drvinceknight replied:
I don't think MWMO is surprising: at worst it'll do as good as the best memory one strategy right? So actually this is a neat outcome I would argue as it's confirming that memory one strategies do do well in N=2. Which in turn is nice when they do poorly for larger populations.

drvinceknight committed:
More analysis and writing:

- Add discussion about numerical inversion;
- Add plots of relative fitness for validation strategy pairs
- Particular section for N=2

Various other changes.

drvinceknight committed:
More analysis and writing:

- Add discussion about numerical inversion;
- Add plots of relative fitness for validation strategy pairs
- Particular section for N=2

Various other changes.

drvinceknight committed:
Update data.

drvinceknight committed:
Update data.

drvinceknight committed:
Make possible to get p_1 when p_N/2 incomplete.

drvinceknight committed:
Make possible to get p_1 when p_N/2 incomplete.

drvinceknight committed:
Add some discussion and references.

drvinceknight committed:
Add some discussion and references.

drvinceknight committed:
Minor reorganisation of files.

drvinceknight committed:
Minor reorganisation of files.

drvinceknight committed:
Add discussion to resistance section.

drvinceknight committed:
Add discussion to resistance section.

drvinceknight committed:
Add tables showing rank vs size.

drvinceknight committed:
Add tables showing rank vs size.

drvinceknight committed:
Minor adjustments and run with new data.

drvinceknight committed:
Minor adjustments and run with new data.

drvinceknight committed:
Rerun with data for N=11, i=1

drvinceknight committed:
Rerun with data for N=11, i=1

Nikoleta-v3 committed:
Add literature.

An introduction to the PD.
Two sentences about emergence of cooperation study
and two sentences about the ZD.

Nikoleta-v3 committed:
Merge branch 'vk' of github.com:Axelrod-Python/axelrod-moran into vk

Nikoleta-v3 committed:
Vine corrections.

I have also included my name in the authors. Thank you once again
for the opportunity.

I decided to sign my name as 'Nikoleta E. Glynatsi` from now on

drvinceknight committed:
Merge pull request #18 from Nikoleta-v3/nik

Nik
